lib/formtools/render-overview.js

```diff
@@ -0,0 +1,88 @@
+
+var async = require('async'),
+    cellRenderers = require('./renderers-cell');
+
+
+var mapCellRenderer = function (type) {
+
+    //add text, password, enum and hidden function in renderers-cell.js
+
+    var cellrenderer;
+
+    switch (type) {
+
+        case 'email':
+            cellrenderer = 'text';
+            break;
+
+        case 'number':
+            cellrenderer = 'text';
+            break;
+
+        case 'digit':
+            cellrenderer = 'text';
+            break;
+
+        case 'tel':
+            cellrenderer = 'text';
+            break;
+
+        case 'documentarray':
+            cellrenderer = 'documentArray';
+            break;
+
+        case 'datetime':
+            cellrenderer = 'dateTime';
+            break;
+
+        case 'datetimeLocal':
+            cellrenderer = 'localDateTime';
+            break;
+
+        default:
+            cellrenderer = 'default';
+            break;
+
+    }
+
+    return cellrenderer;
+};
+
+var generateViewFromModel = exports.generateViewFromModel = function (schema, schemaFields, record, model, callback) {
+
+    var fieldsets = {};
+
+    async.eachSeries(Object.keys(schemaFields), function (fieldName, cb) {
+
+        var field = schemaFields[fieldName],
+            fieldDetail = {
+                label: field.label || ''
+            };
+
+        var title = field.fieldset || 'Summary';
```

Comment:
This bit of logic is not correct. What you are programmed here is if a fieldset is not provided, it will default to summary. That means any fields that doesn't have a fieldset will be grouped into this 'Summary' fieldset. According to how the edit handler works is any fields with no fieldset will be add to the bottom of the form with no grouping. The overview so behave the same as the form edit handler.